### ISSUE TITLE
Building phono3py with openmp

### ISSUE THREAD

**Issue by atztogo** [conda-forge/phono3py-feedstock#25]
I am a developer of phono3py and considering to release it someday soon. The `setup.py` script was updated by the PR (https://github.com/phonopy/phono3py/pull/59). By this change, the installation of phono3py with enabling openmp requires making a file named `site.cfg` on the same directory as that `setup.py` exists. As far as I tested on conda-forge environments of linux and macOS (intel & arm), the following minimum setting of `site.cfg` succeeded to build phono3py with openmp:
```
[phono3py]
extra_compile_args = -fopenmp
```
So I think this has to be done in the conda-forge recipe but I have no idea how to do it. Could you help me?

I wrote some documentation of how to install phono3py as follows.
https://phonopy.github.io/phono3py/install.html#installation-from-source-code
https://phonopy.github.io/phono3py/install.html#installation-instruction-of-latest-development-version-of-phono3py

**Comment by jan-janssen** [comment]
Hi @atztogo , 

We can just include the file in the recipe directory. Can you release the new version on pip, then conda-forge automatically opens a pull request and we can take a look?

**Comment by jochym** [comment]
I second it @atztogo . The simplest way to do it is to release to pypi. The c-f system will pick up the release and probably fail to build. We will fix the scripts then. Unless this is also valid for the currently released version, this is a correct way to do it. Otherwise, it would brake the current package.

**Comment by atztogo** [comment]
Thanks @jan-janssen and @jochym for your quick answers. I will release phono3py in the usual way.
I will make phono3py's `setup.py` fail when `site.cfg` is not found.

**Comment by atztogo** [comment]
I have released v2.2.0 (https://pypi.org/project/phono3py/2.2.0/).
On github action with gcc and openblas for the CI test, I needed to prepare `site.cfg` of
```
[phono3py]
extra_compile_args = -fopenmp
extra_link_args = -lgomp
```

**Comment by atztogo** [comment]
This has been done successfully. Thanks @jan-janssen and @jochym.
